@navikt/core/react/src/form/file-upload/Wrapper.tsx

```diff
@@ -0,0 +1,105 @@
+import cl from "clsx";
+import React, { CSSProperties, MutableRefObject, Ref, useRef } from "react";
+import { ErrorMessage, Label, BodyShort } from "../../typography";
+
+interface Props {
+  label: string;
+  description: string | undefined;
+  className: string | undefined;
+  dropzoneClassName?: string;
+  onDragEnter?: () => void;
+  onDragEnd?: () => void;
+  divRef: any;
+  dropzoneRef?: MutableRefObject<HTMLDivElement | null>;
+  style?: CSSProperties;
+  error: string | undefined;
+  isDraggingOver: boolean;
+  children: React.ReactNode;
+  inputId: string;
+  multiple: boolean;
+  accept: string | undefined;
+  handleUpload: any;
+  fullWidth: boolean;
+}
+
+const Wrapper = ({
+  className,
+  label,
+  description,
+  dropzoneClassName,
+  onDragEnter,
+  onDragEnd,
+  divRef,
+  dropzoneRef,
+  style,
+  error,
+  isDraggingOver,
+  children,
+  inputId,
+  multiple,
+  accept,
+  handleUpload,
+  fullWidth
+}: Props) => {
+  const errorId = `${inputId}-error`
+  const ariaDescribedby = error ? errorId : undefined
+
+  return <div
+    className={cl("navds-form-field", className)}
+    ref={divRef}
+  >
+    <Label
+      htmlFor={inputId}
+      className={cl("navds-form-field__label")}
+    >{label}</Label>
+    {!!description && (
+      <BodyShort
+        className={cl("navds-form-field__description")}
+        as="div"
+      >{description}</BodyShort>
+    )}
+    <div
+      onDragOver={onDragEnter}
+      onDragLeave={onDragEnd}
+      onDragEnd={onDragEnd}
+      onDrop={onDragEnd}
+      style={{
+        width: style?.width ? style.width : fullWidth ? "100%" : "auto",
+        height: style?.height,
+        minWidth: style?.minWidth
```

Comment:
Utforsket litt alternative løsninger for utfordringen med å beholde høyde ved drag-over. Enkleste løsning jeg fant er å vise et `:after`-element over dropzone slik som f.eks [Mesh har gjort her](https://www.meshdesignsystem.com/components/file-upload-component). Slipper da å blande inn ekstra js-logikk for å sette høyde og kan løses med noen linjer CSS 🤞